cmd/utils/utils.go

```diff
@@ -169,3 +169,13 @@ func GetCmdUsage(program, cmd, subcmd, arg string) string {
 		"  " + program + " " + cmd + " " + subcmd + "(s) " + arg + "\n\n"
 	return showCmdUsage
 }
+
+func GetProgramName() string {
+	var programName = os.Args[0]
+	// Replace the dynamic program name with the poject name(mi) to generate bash completion scripts
+	// when the user use go run tools/completeion.go
+	if strings.Contains(programName, "completion") {
```

Comment:
This not very readable and not good to have a completion specific logic in runtime code.

Reply:
But we need to make use of Cobra's code completion functionality as well. Dynamically retrieving the program name from `os.Args[0]` does not work in the case of 'go build

Reply:
Fixed in 80277ce